packages/perseus/src/jipt-paragraphs.js

```diff
@@ -11,22 +11,28 @@ const arrayRules: ParserRules = {
     fence: {
         match: SimpleMarkdown.defaultRules.fence.match,
         order: 1,
-        // $FlowFixMe[incompatible-type]
-        parse: (capture, state, parse) => capture[3],
+        parse: (capture, state, parse) => ({
+            type: "codeBlock",
+            lang: capture[2] || undefined,
+            content: capture[3],
+        }),
     },
     paragraph: {
         match: SimpleMarkdown.defaultRules.paragraph.match,
         order: 2,
-        // $FlowFixMe[incompatible-type]
-        parse: (capture, state, parse) => capture[1],
+        parse: (capture, state, parse) => ({
+            content: capture[1],
+        }),
     },
```

Comment:
So this is the change that actually fixed the issue and the changes to simple-markdown will make it easier to identify similar bugs in the future?

Reply:
Yes. I thought it would be useful to have the assertion in SimpleMarkdown until we have more complete Flow types in place (and even then, we could leave it in there without harming things)